gcc/rust/rust-session-manager.cc

```diff
@@ -474,6 +526,28 @@ Session::enable_dump (std::string arg)
 void
 Session::parse_files (int num_files, const char **files)
 {
+  rust_assert (num_files > 0);
+
+  if (options.crate_name.empty ())
+    {
+      /* HACK: We use the first file to infer the crate name, which might be
+       * incorrect: since rustc only allows one file to be supplied in the
+       * command-line */
+      auto crate_name = infer_crate_name (files[0]);
+      rust_debug_loc (Location (), "inferred crate name: %s",
```

Comment:
you can call rust_debug instead of rust_debug_loc when you dont have a location

Reply:
> you can call rust_debug instead of rust_debug_loc when you dont have a location

I see. I will fix this.